lib/assessment.rb

```diff
@@ -5,7 +5,12 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.reduce do |item, memo|
+      if memo.to_s.length > item.to_s.length
```

Comment:
We don't know that these aren't strings already. All we know is that whatever kind of data is found in `enum`, it has a `.length` method.